common/src/main/scala/explore/implicits.scala

```diff
@@ -43,6 +44,33 @@ trait ListImplicits {
       monoid: Monoid[Out]
     ): Out = hlists.map(_.map(singleton)).combineAll
   }
+
+  implicit class ViewListOps[F[_], A](val viewList: ViewF[F, List[A]]) {
+    def toAgGridData[B](eqBy: A => B)(implicit eq: Eq[B]): List[ViewF[F, A]] =
```

Comment:
Should probably have better names for these. Would calling them `sequence` and `sequenceSorted` be more clear or more confusing?

Reply:
`sequence` could be confusing considering its meaning in `cats`